api/client/service.proto

```diff
@@ -45,3 +46,34 @@ service NighthawkService {
   rpc ExecutionStream(stream ExecutionRequest) returns (stream ExecutionResponse) {
   }
 }
+
+// Encapsulates an ExecutionResponse.
+message StoreExecutionRequest {
+  // Response contains the effective execution id, which will serve as the lookup key.
+  ExecutionResponse execution_response = 1;
+}
+
+// Empty return value message, that serves as a future extension point.
+message StoreExecutionResponse {
+}
+
+message SinkRequest {
+  // Unique id for lookup purposes.
+  string execution_id = 1;
+}
+
+message SinkResponse {
+  // Response associated to the requested execution id.
+  ExecutionResponse execution_response = 1;
+}
+
+service NighthawkSink {
```

Comment:
Would you mind moving the sink related service and its messages to a separate proto file for clarity? Say `sink_service.proto`?

Reply:
Done in fe2eac357d600922489e611a8843548b544a16dd. Let me know what you think, happy to iterate more on this.